src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/data/EnvelopeSummaryRepositoryTest.java

```diff
@@ -0,0 +1,188 @@
+
+        // after the request date
+        envelopeRepository.insert(new NewEnvelope(CONTAINER_2, FILE_2_4, createdAt8, dispatchedAt, Status.REJECTED));
+        eventRecordRepository.insert(new NewEventRecord(CONTAINER_2, FILE_2_4, FILE_PROCESSING_STARTED));
+
+        // when
+        List<EnvelopeSummary> result = repo.find(
+            LocalDate.parse("2019-12-20").atStartOfDay().toInstant(UTC),
+            LocalDate.parse("2019-12-21").atStartOfDay().toInstant(UTC)
+        );
+
+        // then
+        assertThat(result.stream()
+                       .collect(toList()))
+            .usingFieldByFieldElementComparator()
+            .containsExactlyInAnyOrder(
+                new EnvelopeSummary(CONTAINER_1,
+                                    FILE_1_2,
+                                    createdAt2,
+                                    dispatchedAt,
+                                    Status.DISPATCHED,
+                                    false,
+                                    DISPATCHED,
+                                    null,
+                                    lastEvent12CreatedAt
+                ),
+                new EnvelopeSummary(CONTAINER_1,
+                                    FILE_1_3,
+                                    createdAt3,
+                                    dispatchedAt,
+                                    Status.DISPATCHED,
+                                    true,
+                                    DELETED,
+                                    null,
+                                    lastEvent13CreatedAt
+                ),
+                new EnvelopeSummary(CONTAINER_2,
+                                    FILE_2_1,
+                                    createdAt4,
+                                    null,
+                                    Status.REJECTED,
+                                    false,
+                                    DUPLICATE_REJECTED,
+                                    "Duplicate",
+                                    lastEvent21CreatedAt
+                ),
+                new EnvelopeSummary(CONTAINER_2,
+                                    FILE_2_2,
+                                    createdAt5,
+                                    null,
+                                    Status.REJECTED,
+                                    false,
+                                    DELETED_FROM_REJECTED,
+                                    null,
+                                    lastEvent22CreatedAt
+                ),
+                new EnvelopeSummary(CONTAINER_2,
+                                    FILE_2_3,
+                                    createdAt6,
+                                    dispatchedAt,
+                                    Status.DISPATCHED,
+                                    false,
+                                    null,
+                                    null,
+                                    null
+                )
```

Comment:
first argument should be new line as well.

on other note: so many fields in assertion 🙊